web/menu.jspf

```diff
@@ -117,7 +121,11 @@ org.opensolaris.opengrok.web.Util"
         if (ph.getGroups().size() > 0 && ph.getAllUngrouped().size() > 0) {
             %><optgroup label="Other"><%
         }
-        for ( Project p : ph.getAllUngrouped()) {
+        for (Project p : ph.getAllUngrouped()) {
```

Comment:
I wondered whether changing `ProjectHelper#cacheProjects()` to suppress the projects would be good idea (to make the counts on the index page next to the groups reflecting only indexed projects) however changing the caching might have undesired effects.